docs/command-reference/acl/deluser.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Delete a user
+---
+
+# ACL DELUSER
+
+## Syntax
+
+    ACL DELUSER username
+
+**ACL categories:** @admin, @slow, @dangerous
```

Comment:
wow now we need to add ACL categories to all of our commands right?

Reply:
yes this is what I am doing all day :rofl: